From 623f471f2c1958fc8a81adb58a51611476ddf2f1 Mon Sep 17 00:00:00 2001
From: Amlana Bhuyan <amlanab@ami.com>
Date: Sat, 28 Oct 2023 23:37:58 +0530
Subject: [PATCH] libvncserver Keyboard LED sync

 Addition of missing function declaration for rfbSendKeyboardLedState()
 required for Keyboard LED sync.

Signed-off-by: Amlana Bhuyan <amlanab@ami.com>
---
 include/rfb/rfb.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/rfb/rfb.h b/include/rfb/rfb.h
index 79a446f1..ecbfe43d 100644
--- a/include/rfb/rfb.h
+++ b/include/rfb/rfb.h
@@ -806,6 +806,7 @@ extern rfbBool rfbSendFramebufferUpdate(rfbClientPtr cl, sraRegionPtr updateRegi
 extern rfbBool rfbSendRectEncodingRaw(rfbClientPtr cl, int x,int y,int w,int h);
 extern rfbBool rfbSendUpdateBuf(rfbClientPtr cl);
 extern void rfbSendServerCutText(rfbScreenInfoPtr rfbScreen,char *str, int len);
+extern rfbBool rfbSendKeyboardLedState(rfbClientPtr cl);
 #ifdef LIBVNCSERVER_HAVE_LIBZ
 extern void rfbSendServerCutTextUTF8(rfbScreenInfoPtr rfbScreen,char *str, int len, char *fallbackLatin1Str, int latin1Len);
 #endif
-- 
2.25.1

